docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.module

```diff
@@ -31,3 +33,30 @@ function hs_paragraph_types_preprocess_field__field_hs_collection_items(&$variab
   $variables['attributes']['class'][] = "item-per-row--$items_per_row";
   $variables['#attached']['library'][] = 'hs_paragraph_types/hs_collection';
 }
+
+/**
+ * Implements hook_field_widget_single_element_WIDGET_TYPE_form_alter().
+ */
+function hs_paragraph_types_field_widget_single_element_paragraphs_form_alter(&$element, &$form_state, $context) {
+  // Change the paragraph type label for hs_hero_image embedded in hs_carousel.
+  if ($element['#paragraph_type'] === 'hs_hero_image' && $element['#bundle'] === 'hs_carousel') {
```

Comment:
No need for the triple `=` here. Double is enough and is consistent with out normal code practice

Reply:
Fixed